tests/testExport.py

```diff
@@ -262,6 +262,13 @@ def test_saveImage_multipage_keywordArgument(self):
                 drawBot.saveImage(tmp.path, multipage=False)
         self.assertEqual(output.lines(), [])
 
+    def test_unevenPages_mp4(self):
+        self.makeTestAnimation(pageWidth=500, pageHeight=501)
```

Comment:
We should also test odd width and even height. Perhaps just generate a single frame?

Reply:
This is actually tested in other tests where an mp4 is generated with even dimensions.

Reply:
I meant the other way: odd height, even width.